lib/ReactViews/ExplorerWindow/Tabs/MyDataTab/AddData.jsx

```diff
@@ -118,18 +123,21 @@ const AddData = createReactClass({
           message: `An error occurred trying to add data from URL: ${url}`
         });
         newItem.setTrait(CommonStrata.user, "url", url);
-        promise = newItem.loadMetadata().then(() => newItem);
+        promise = newItem
+          .loadMetadata()
+          .then(result => result.throwIfError() && newItem);
```

Comment:
Does this error get handled anywhere?

Reply:
Yep - it is a bit weird but it gets handled in `addUserCatalogMember`.

I have refactored it to make it a bit more clear

Reply:
Whoops, I misread `addUserCatalogMember` as the definition of the next function, not a call to a function that then handles the errors.